libs/domain/content/link/link.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host([dropdown]) {
+    --oryx-button-color: var(--oryx-color-neutral-12);
+
+    padding: 9px 15px;
+  }
+
+  :host([dropdown]:hover) {
```

Comment:
If you need just and attribute for stylisation, try to toggle it inside `elementEffect`, or composition supports hover query, check the **home-page.ts** for some examples

Reply:
Or we can just apply that stylisation directly to the button, ie:
```
oryx-button[type="text"] {
...
}
```